state-chain/custom-rpc/src/lib.rs

```diff
@@ -213,19 +213,48 @@ pub struct RpcAuctionState {
 	min_active_bid: Option<NumberOrHex>,
 }
 
+#[derive(Serialize, Deserialize)]
+#[serde(rename_all = "SCREAMING_SNAKE_CASE")]
+pub enum SwapFeeKind {
+	Network,
```

Comment:
It's a commonly used format to represent enum values as strings

Reply:
Sure, but it seems a bit arbitrary to introduce this convention here when we don't use it elsewhere (think asset enums, chain enums, etc).

Reply:
I think the asset and chain enums are external identifiers that we happen to model using enums. Here we have our own enum that we want to represent externally which allows us some leeway in the serialization. I can change it to be something else but I didn't see much in the way of convention for the serialization of value-less enums in the custom RPCs

Reply:
I don't think the serialization of enums in events plays a role here because it's an entirely different kind of API. Since this is explicitly a JSON API, it makes sense to me to build an API that I would expect to consume when calling a JSON API, which includes `snake_case` or `SCREAMING_SNAKE_CASE` for enum values when they are represented as strings. I don't think there's anything special about `SwapFeeKind`, I've just never seen enums represented as camel-cased string values and so I just default to building an API that would feel familiar to me. I've reverted the change as it's not the convention for the custom RPCs.

Reply:
Thanks - not questioning whether or `SCREAMING_SNAKE_CASE` is or isn't more idiomatic in the JSON Api world, just pointing out that we follow other conventions (for better or worse) elsewhere in our api. 

I mentioned events because they are frequently represented as JSON in polkaJS and in the explorer etc. and to my knowledge it also defaults to camel case because it follows rust conventions. It's not unreasonable to assume people will expect to see the same values in the block explorers as in the rpcs.